### PR TITLE
updated index page

### PR DESCRIPTION
**Pull request by analoo**

### CODE REVIEW

File: src/main/resources/messages.properties [hunk 1]
```diff
@@ -37,44 +37,23 @@ language-preferences.english=English
...
 
-index.title=Louisiana Document Uploader
-index.header=Upload your benefits documents easily online.
+index.title=Louisiana Digital Assister
+index.header=Apply for food benefits easily online
 index.louisiana-partner=Official State of Louisiana partner.
-index.upload-description=Upload documents by phone or computer in 5 minutes or less.
-index.upload-documents-primary=Upload documents
+index.upload-description=Apply for SNAP benefits in about 20 minutes or less.
+index.upload-documents-primary=Apply Now
```
Comment (sree-cfa) [inline src/main/resources/messages.properties:44]: There should be an arrow on the button on the index page "Apply Now -->"

File: src/main/resources/messages.properties [hunk 2]
```diff
@@ -37,44 +37,23 @@ language-preferences.english=English
...
-index.fitap.description=<b>FITAP</b> (Family Independence Temporary Assistance Program) provides cash assistance to help people meet their basic needs until they are able to support themselves.
-index.kscp.description=<b>KCSP</b> (Kinship Care Subsidy Program) also provides cash assistance to relatives who are taking care of children who are unable to live with their parents. Eligible relatives can receive monthly cash payments to help cover the costs of caring for the child, such as housing, food, clothing, and other basic needs.
-index.looking-for-help=<b>Looking for assistance with other programs?</b>
-index.visit-dcfs-website=Visit <a target="_blank" rel="noreferrer noopener" href="https://www.dcfs.louisiana.gov">the Department of Children and Family Services website</a> for more information.
+index.upload-documents-second=Aplique ya<i class="button__icon icon-arrow_forward"></i>
+index.upload-documents-second.link=/flow/laDigitalAssister/parish?lang=es
+index.upload-documents-third=\u0110\u0103ng k\u00FD ngay<i class="button__icon icon-arrow_forward"></i>
+index.upload-documents-third.link=/flow/laDigitalAssister/parish?lang=vi
```
Comment (sree-cfa) [inline src/main/resources/messages.properties:52]: just fyi - these are the properties re Karens comment on the Asana ticket. Just need to update them in the messages_es and messages_vi property files too

Comment (sree-cfa) [reply src/main/resources/messages.properties:52]: Actually - might be better to change it in transifex

Comment (analoo) [reply src/main/resources/messages.properties:52]: Thanks for flagging these sree. I went ahead and fixed them

**Review by sree-cfa — APPROVED** [review]
lgtm

**Pull request merged**